### PR TITLE
new homework

### PR DESCRIPTION
**Pull request by lototska**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
+    <body>
+        <h1></h1>
+        <ul>
+            <li><a href="#yak1">lifestyle</a></li>
+            <li><a href="#yak2">photodiary</a></li>
+            <li><a href="#yak3">music</a></li>
+            <li><a href="#yak4">travel</a></li>
+        </ul>
+        <div id="yak1"></div>
+        <img src="img/image.1jpg" width="270" height="200" alt="фото хлопця який танцює">
```
Comment (yaroslav-lototskyi) [inline index.html]: only English

File: index.html [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
-   <h1>Hello World</h1> 
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+    </head>
+    <body>
+        <h1></h1>
```
Comment (yaroslav-lototskyi) [inline index.html]: for what empty tag ?

File: index.html [hunk 3]
```diff
@@ -1,12 +1,39 @@
...
+        <ul>
+            <li><a href="#yak1">lifestyle</a></li>
+            <li><a href="#yak2">photodiary</a></li>
+            <li><a href="#yak3">music</a></li>
+            <li><a href="#yak4">travel</a></li>
+        </ul>
+        <div id="yak1"></div>
```
Comment (yaroslav-lototskyi) [inline index.html]: for what empty tag ? you must push all content in this tag !!

File: index.html [hunk 4]
```diff
@@ -1,12 +1,39 @@
...
+        <div id="yak3"></div>
+        <img src="img/image3.jpg" width="270" height="200" alt="2 дівчини">
+        <p>photodiary</p>
+        <h3>American dream</h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco. </p>
+        <div id="yak4"></div>
+        <img src="img/image4.jpg" width="270" height="200" alt="фото хлопця зі спини">
+        <p>photodiary</p>
+        <h3>A day exploring the Alps</h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco lab.</p>
```
Comment (yaroslav-lototskyi) [inline index.html]: wrap all content in tag ```ul```

File: index.html [hunk 5]
```diff
@@ -1,12 +1,49 @@
...
+                <div id="yak4"><img src="img/image4.jpg" width="270" height="200" alt="photo of a guy from the back">
+                    <p>photodiary</p>
+                    <h3>A day exploring the Alps</h3>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco lab.</p>
+                </div>
+            </li>
+        </ul>
+        <button>load more</button>
+    </body>
 </html>
```
Comment (yaroslav-lototskyi) [inline index.html:49]: add new line at end of file

File: index.html [hunk 6]
```diff
@@ -1,12 +1,49 @@
...
+            <li><a href="#yak1">lifestyle</a></li>
+            <li><a href="#yak2">photodiary</a></li>
+            <li><a href="#yak3">music</a></li>
+            <li><a href="#yak4">travel</a></li>
+        </ul>
+        <ul style="list-style: none">
+            <li>
+                <div id="yak1"><img src="img/image.1jpg" width="270" height="200" alt="photo of a guy dancing">
+                    <p>Lifestyle</p>
+                    <h2>More than just a music festival</h2>
```
Comment (yaroslav-lototskyi) [inline index.html:20]: ident

**Comment by lototska** [comment]
Homework 3
https://lototska.github.io/hw-html/ git page
https://github.com/lototska/hw-html/pulls code source


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Без
вирусов. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#m_-2571551951388177552_m_5450529003431443378_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

ср, 14 квіт. 2021 о 13:36 Yaroslav Lototskyi ***@***.***>
пише:

**Pull request merged**